v1.0/v1.0/writable-dir-docker.cwl

```diff
@@ -0,0 +1,22 @@
+cwlVersion: v1.0
+
+class: CommandLineTool
+
+requirements:
+  - class: InlineJavascriptRequirement
+  - class: InitialWorkDirRequirement
+    listing:
+      - entryname: emptyWritableDir
+        entry: "$({class: 'Directory', listing: []})"
+        writable: true
+
+hints:
+  - class: DockerRequirement
+    dockerPull: alpine
+
```

Comment:
This is great, thank you! Please add a proof that the directory is truly writeable.